server.js

```diff
@@ -188,9 +186,56 @@ app.get('/:username/:plantId', async (req, res) => {
 });
 
 
+// Edit plant profile
+app.patch('/:username/:plantId/edit', authenticateUser);
+app.patch('/:username/:plantId/edit', async (req, res) => {
```

Comment:
I'm not sure you need a new address for a patch call, my understanding is that you can have 
`app.get('/:username/:plantId/', authenticateUser)`
`app.post('/:username/:plantId', authenticateUser)`
`app.patch('/:username/:plantId', authenticateUser)`

Reply:
Good point! I'll update this in master